lib/positions.js

```diff
@@ -89,7 +114,7 @@ exports.listPositionsWithApprovedCandidates = async (req, res) => {
             .filter((candidate) => candidate.status !== 'rejected')
             .map((candidate) => {
                 if (candidate.status === 'approved') {
-                    return helpers.blacklistObject(candidate.toJSON(), ['email']); // the email should be visible to JC only
+                    return helpers.blacklistObject(candidate.toJSON(), ['email', 'notification_email']); // the email should be visible to JC only
```

Comment:
Why do you use both the old and the new everywhere updated notificiation email here?

Reply:
Older applications still use the email stored in the database, I want that neither are displayed to others

Reply:
Okay sounds good